systemd.ml

```diff
@@ -52,4 +52,18 @@ module Daemon = struct
 
   let listen_fds_lwt () =
     List.map Lwt_unix.of_unix_file_descr (listen_fds ())
+
+  let get_args () =
+    [
+      ("-loglevel", Arg.String Log.set_loglevels, " ([<facil|prefix*>=]debug|info|warn|error[,])+");
+      ExtArg.may_str "logfile" Daemon.logfile "<file> Log file";
+      "-runas",
```

Comment:
technically it is weird to see runas in systemd module, user to run as is also managed by systemd, no need to handle in the app. logfile is debatably similar but it makes sense to log from the app (for logrotate with file to work)

Reply:
Sure, removed `-runas` in https://github.com/ahrefs/devkit/pull/20/commits/240cd111f114501b89a188d2cf0e975ff1b591a6.